### PR TITLE
images replaced on KMSv2 blog article with SVG vector versions  #36654

### PR DESCRIPTION
**Pull request by prsndp**
On publication, https://k8s.io/blog/2022/09/09/kms-v2-improvements/ Mermaid generated online images replaced using figures in the markdown source and used SVG format of images.  #36654

### CODE REVIEW

File: content/en/blog/_posts/2022-08-03-kms-v2-alpha.md [hunk 1]
```diff
@@ -32,9 +32,11 @@ The KMS v2 enhancement attempts to address all of these shortcomings, though not
...
+
+{{< figure src="/images/blog/2022-09-09-kms-v2-improvements/kms-v2-improvements-encrypt-request.svg" alt="KMS V2 Improvements" class="release-logo" >}}
```
Comment (sftim) [inline content/en/blog/_posts/2022-08-03-kms-v2-alpha.md:36]: The HTML class we use here should _not_ be `release-logo`. Try `diagram-large` instead.

File: content/en/blog/_posts/2022-08-03-kms-v2-alpha.md [hunk 2]
```diff
@@ -32,9 +32,11 @@ The KMS v2 enhancement attempts to address all of these shortcomings, though not
...
+{{< figure src="/images/blog/2022-09-09-kms-v2-improvements/kms-v2-improvements-decrypt-request.svg" alt="KMS V2 Improvements" class="release-logo" >}}
```
Comment (sftim) [inline content/en/blog/_posts/2022-08-03-kms-v2-alpha.md:39]: The HTML class we use here should _not_ be `release-logo`. Try `diagram-large` instead.

Comment (prsndp) [reply content/en/blog/_posts/2022-08-03-kms-v2-alpha.md:39]: @sftim okay, let me update it.

**Comment by prsndp** [comment]
> This PR deletes the API reference.
> 
> We shouldn't merge this as-is.
> 
> /hold
> 
> @prsndp can you see how to fix this?

@sftim, I am creating a fork of this repo again I will fix it.

**Pull request closed without merging**